### DISCUSSION TITLE
Is there a way to set Alarms when your RBMK Reactor is about to blow up or has Critical Status?

### DISCUSSION THREAD

**Discussion by diyamomd45** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1048] [Q&A]
Hello, I was wondering if there is a way to set Alarms or Warnings when a Reactor (A RBMK Reactor) is in critical Status?
I saw a red thing behind the RBMK Console but im not sure if its a Redstone Output. 
Is there a way to link it with CC or OC?

**Comment by BallOfEnergy1** [comment]
Using the OpenComputers Integration you can read the heat of a rod, column, or other RBMK component, and then do something based on the results. (Wiki page coming soon on OpenComputers and NTM.)
